packages/textlint-tester/src/textlint-tester.ts

```diff
@@ -313,6 +319,7 @@ valid: [ "text", { text: "text" }, { inputPath: "path/to/file" } ]
         const text = invalid.text;
         const ext = invalid.ext !== undefined ? invalid.ext : ".md";
         const options = invalid.options !== undefined ? invalid.options : undefined;
+        const description = invalid.description !== undefined ? invalid.description : undefined;
```

Comment:
```suggestion
        const description = invalid.description;
```
It is same meaning.

Reply:
Ah `options` did same thing.

Reply:
Do you think options should be fixed in this PR?

Reply:
yes please.
It will be same result.